src/super_gradients/training/datasets/dali_datasets/dali_dataloaders.py

```diff
@@ -0,0 +1,18 @@
+import collections
+
+
+class DaliClassificationDataLoader(collections.Iterator):
```

Comment:
@shaydeci - I think a more suitable location for this class is inside dataset_interface, since this is not a dataset for DL training... Let's move this please